# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1402072193 14400
#      Fri Jun 06 12:29:53 2014 -0400
# Branch stable
# Node ID 48828394487db51fc5ccfc27a00e4abdca880324
# Parent  492c5614953571a0b07f4c1621cc7bc5c85c0251
Added tag rc-3-8-2-1 for changeset 492c56149535

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -82,8 +82,9 @@ 3a9efb68272df556dccb84c87933dd8238e88902
 cc13924a4266fb0359f59fabdce11071e6051d48 ss-3-7-7
 d734216aa2b1022c190e6b07d7ccdd59fe15678f rc-3-8-0-1
 f1fa148766d623f7f24cf541ea8b88d8c1b63d33 rc-3-8-0-2
 b72bcf5f78cc2231e9bf2735d3b8d92779fc4dff release-3-8-0
 85d31344fb23725791bc687337295d47416b9c17 rc-3-8-1-2
 02ce68d63fba31cd14ac060c087212a032ae3202 rc-3-8-1-3
 dd669c2ae76c167613f54d6f4db0130fa2124cac rc-3-8-1-4
 43cc202335dc4a53a3d8fc9ca90acaa81d2e63d3 release-3-8-1
+492c5614953571a0b07f4c1621cc7bc5c85c0251 rc-3-8-2-1
